composer.json

```diff
@@ -9,15 +9,8 @@
     }
   ],
   "type": "wordpress-plugin",
-  "repositories": [
-    {
-      "type": "vcs",
-      "url": "git@github.com:OpenWebconcept/plugin-pdc-base.git"
-    }
-  ],
   "require": {
-    "php": ">=7.0",
-    "plugin/pdc-base": "^3.0.0"
+    "php": ">=7.0"
```

Comment:
Denk dat het ook wel handig is om gelijk php 8 hier aan toe te voegen. EOL van PHP7 is in november dacht ik. Wat jij?

Reply:
Daar heb je wel een punt ja. Aan de andere kant: WordPress zelf is nog niet eens zo ver dat ze 8.0 volledig ondersteunen :( 

Compromis: `^7.4|^8.0` en dan 7.4 droppen zodra WordPress volledig 8.0 compatible is? [Zo doe ik het ook met mijn plugin](https://github.com/OpenWebconcept/open-government-publications/blob/master/composer.json#L26)

Reply:
Ja dat was precies wat ik bedoelde idd, maar ik kwam wel tegen net op het web dat Wordpress wel php 8.1 ondersteund. Waar kwam jij het tegendeel tegen?

Reply:
Van Engels weer naar Nederlands, maar goed :-). Ik zie dat PHP 7.4 nog tot in november 2022 updates krijgt. Had gedacht dat dit langer zou zijn. Laten we gewoon de minimale PHP versie van WordPress aanhouden. Van wat ik heb gehoord en gezien is PHP 8 ondersteuning binnen WordPress en de plug-ins nog verre van optimaal.

Reply:
Minimale PHP versie voor WordPress 6.0 [is nog steeds 5.6](https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/) 😩 Dat vind ik een te grote stap terug. Zowel wat betreft language features als security. 

WP is compatible met PHP 8+ in de zin van: het werkt. Maar als je error reporting instelt wordt/werd je overladen met (deprecation) warnings vanuit de WP core.

Reply:
Maar ze raden 7.4+ aan. Wat mij betreft hanteren we 7.4 als minimale versie. Als bij 8 met error_reporting je overladen wordt met dergelijke meldingen zorgt dat voor enorm veel ruis in logfiles e.d. en dan zou ik zelfs willen zeggen 7.4 als maximale versie. Al heeft het wat mij betreft niet zoveel zin om een maximale versie af te dwingen.

Kijkende naar je link wordt PHP 8.0 nog niet voldoende genoeg ondersteund door WordPress en daarmee reden genoeg om 8.0 niet als minimale versie te gebruiken.

Reply:
Mee eens maar daarom is `^7.4|^8.0` toch een goede oplossing?